Gems/Atom/RPI/Code/Include/Atom/RPI.Reflect/Model/ModelAssetHelpers.h

```diff
@@ -28,6 +69,14 @@ namespace AZ
             //! @param modelAsset An empty modelAsset that will get filled in with unit X data.
             static void CreateUnitX(ModelAsset* modelAsset);
 
+
+            template<typename T>
+            static size_t GetAlignedCount(size_t vertexCount, AZ::RHI::Format bufferFormat, uint32_t alignment);
```

Comment:
Can you add comment about each of the input parameters? 
I have trouble to understand what the bufferFormat, alignment, T are just by looking at the function declaration.